src/base/bittorrent/session.cpp

```diff
@@ -2090,7 +2090,8 @@ bool Session::addTorrent(QString source, const AddTorrentParams &params)
     if (magnetUri.isValid()) {
```

Comment:
I would remove braces from this "if" body.

Reply:
Yes agreed, removing the braces